scripts/services/check-removed-services.js

```diff
@@ -0,0 +1,100 @@
+const { promises: fs } = require('fs');
+const path = require('path');
+const yaml = require('js-yaml');
+
+const servicesDir = path.resolve(__dirname, '../../services/');
+
+const checkRemovedServices = async (distFolder, servicesJSON) => {
+    // changes names of services to lower case without, and all - change to _
+    const normalizeFileName = (serviceName) => {
+        const specificCharacters = new RegExp(/[^a-z0-9.]/, 'g');
+        const lowerCased = serviceName.toLowerCase();
+        const replaceSpecialCharacters = lowerCased.replace(specificCharacters, '');
+        return replaceSpecialCharacters;
+    };
+    // get array with old services objects and check if json is valid
+    const getServicesData = async (servicesFile) => {
+        let OldServicesDataArray = [];
+        try {
+            // get json data
+            const OldDataJSON = await fs.readFile(servicesFile);
+            // form json to object
+            const OldDataObj = JSON.parse(OldDataJSON);
+            // get only services array
+            OldServicesDataArray = OldDataObj.blocked_services;
```

Comment:
no need for `OldServicesDataArray`:
- `OldDataObj.blocked_services` can be returned here without assigning
- otherwise better to return `null` explicitly (because of the `!oldServicesData` condition below)

```
const getServicesData = async (servicesFile) => {
    try {
        const oldDataJSON = await fs.readFile(servicesFile);
        const oldDataObj = JSON.parse(oldDataJSON);
        // FIXME: question: should we check that oldDataObj.blocked_services is an array?
        return oldDataObj.blocked_services;
    } catch (error) {
        console.error('Error while reading JSON file:', error);
        return null;
    }
};

const oldServicesData = await getServicesData(servicesJSON);

if (!oldServicesData) {
    return;
}
```